test/test_browser.py

```diff
@@ -3809,14 +3809,23 @@ def test_pthread_main_thread_blocking(self, name):
     print('Test that we error if not ALLOW_BLOCKING_ON_MAIN_THREAD')
```

Comment:
Can you remove this `print` statement, or turn it into docstring?

Reply:
33468ae7f819b7129882495f59077924b94cf018